python_modules/libraries/dagster-aws/dagster_aws/s3/resources.py

```diff
@@ -26,6 +26,12 @@
         is_required=False,
         default_value=5,
     ),
+    "profile_name": Field(
+        str,
+        description="TSpecifies a profile to connect that session",
+        is_required=False,
+        default_value="default",
+    ),
```

Comment:
I don't think its quite right for this bit of config schema to be defined for the base `s3_resource` and ignored.

I think if we want to support this field it should just be used in the `s3_resource`

Reply:
Hi Alex, 

Thanks for your suggestions. 

I will update the PR accordingly.